src/main/webapp/app/admin/system-notification-management/system-notification-management-detail.component.html

```diff
@@ -0,0 +1,23 @@
+<div *ngIf="notification">
+    <h2>
+        <span jhiTranslate="artemisApp.systemNotification.systemNotification">System notification</span> [<b>{{ notification.id }}</b
+        >]
```

Comment:
The line break looks a bit odd.

Reply:
Good catch! Prettier introduced that line break. I split it up in two lines.